GitTfs.Vs2015/paket.references

```diff
@@ -1,6 +1,2 @@
-//If you plan to update this file, you must first absolutly read our documentation on the subject
-//because there is some subtleties on how we use it in our project:
-//https://github.com/git-tfs/git-tfs/blob/master/doc/paket.md
```

Comment:
> How do you add comments in paket.references?

I think @pmiossec just edited the file to add them.

> I removed them to avoid:

If it's causing errors, then don't worry about adding it back in.